Bit-Manipulation/test/SetBit.test.js

```diff
@@ -0,0 +1,21 @@
+import { setBit } from '../SetBit'
+
+test('should set bit at the given bit Position', () => {
```

Comment:
Expand the test message a bit to give as much information as possible.
Here:
```suggestion
test('Set bit number 0 in 1:', () => {
```

Reply:
Add this for all the tests.

Reply:
@AbhinavXT Do not apply the same name to all of them, replace the 0 and 1 with the bit number and the number respectively.

Reply:
Yes, Sorry about that. I will correct the changes.